### DISCUSSION TITLE
A user is logged in as another user

### DISCUSSION THREAD

**Discussion by LujienZ** [requarks/wiki#7387] [Error / Bug Report] [answered]
Hello,

I’m encountering a recurring issue with authentication. After some time (which doesn’t seem fixed), some users (myself included) find themselves logged in as another person.

For example, I, John, have been logged in as John for several days, but suddenly I find myself logged in as Paul.

If I log out and log back in, I return to being John again.

Temporary solution I’ve found: I use the "Generate New Authentication Public / Private Key Certificates" function to force a logout.

Any idea what could be causing the issue?

**Context**
*Version:* 2.5.298
*Authentication method:* SAML 2.0 (but reproducible with local database)
*Token expiration:* 30 minutes
*Token renewal period:* 2 days
*Hosted behind an IIS reverse proxy*

**Comment by NGPixel** [comment, marked as answer]
Disable caching on your IIS reverse proxy.
